build/charts/antrea/crds/flowexporterdestination.yaml

```diff
@@ -0,0 +1,103 @@
+      - name: Address:Port
+        type: string
+        description: Address of flow collector.
+        jsonPath: .spec.address
+      schema:
+        openAPIV3Schema:
+          type: object
+          required:
+            - spec
+          properties:
+            spec:
+              type: object
+              required:
+                - address
+                - protocol
+              properties:
+                address:
+                  type: string
+                  description: >
+                    The flow collector address including port as a string.
+
+                    Example:
+                    - flow-aggregator/flow-aggregator:14739
+                    - 10.244.10.10:4739
+                  pattern: ^.+:[0-9]+$
+                protocol:
+                  type: object
+                  description: >
+                    The protocol used to send flow details.
+
+                    Exactly one must be defined and non-nil.
+                  oneOf:
+                  - required: [ipfix]
+                  - required: [grpc]
+                  properties:
+                    ipfix:
+                      type: object
+                      description: Configuration for using IPFIX protocol.
+                      required:
+                      - transport
+                      properties:
+                        transport:
+                          type: string
+                          enum:
+                          - tcp
+                          - udp
+                          - tls
+                    grpc:
+                      type: object
+                      description: Configuration for using gRPC protocol.
+                filter:
+                  type: object
+                  properties:
+                    protocols:
+                      type: array
+                      description: >
+                        Filter for only flows whose protocol which match this filter.
+                        The default is accept all protocols if unset or nil.
+
+                        Supported values are [tcp, udp, sctp].
+                      items:
+                        type: string
+                        enum:
+                        - tcp
+                        - udp
+                        - sctp
+                activeFlowExportTimeoutSeconds:
+                  type: integer
+                  description: >
+                    Provide the active flow export timeout in seconds, which is the timeout after which a flow
+                    record is sent to the collector for active flows. Thus, for flows with a continuous
+                    stream of packets, a flow record will be exported to the collector once the elapsed
+                    time since the last export event is equal to the value of this timeout.
+                  minimum: 1
+                  default: 5
+                idleFlowExportTimeoutSeconds:
+                  type: integer
```

Comment:
add `format: int32` for both and make them `int32` (instead of `int64`) in `types.go`